scratch_map/countries.py

```diff
@@ -0,0 +1,8 @@
+class Countries:
+    country_list = []
+
```

Comment:
The two below functions are a bit basic but might be necessary later when we add in data validation and stuff